eth_worker/eth_manager/celery_tasks.py

```diff
@@ -23,40 +23,75 @@ def after_return(self, status, retval, task_id, args, kwargs, einfo):
     'retry_backoff': True
 }
 
-processor_task_config = dict(task_config)
-processor_task_config['max_retries'] = 0
+processor_task_config = {
+    **base_task_config,
+    'max_retries': 0,
+    'queue': 'processor'
+}
+
+
+@celery_app.task(**base_task_config)
+def deploy_exchange_network(self, deploying_address):
+    return eth_manager.task_interfaces.composite.deploy_exchange_network(deploying_address)
+
+
+@celery_app.task(**base_task_config)
+def deploy_and_fund_reserve_token(self, deploying_address, name, symbol, fund_amount_wei):
+    return eth_manager.task_interfaces.composite.deploy_and_fund_reserve_token(
+        deploying_address,
+        name, symbol, fund_amount_wei)
 
 
-@celery_app.task(**task_config)
-def create_new_blockchain_wallet(self, wei_target_balance=0, wei_topup_threshold=0):
-    wallet = persistence_interface.create_new_blockchain_wallet(wei_target_balance = wei_target_balance,
-                                                                wei_topup_threshold=wei_topup_threshold)
+@celery_app.task(**base_task_config)
+def deploy_smart_token(self, deploying_address,
+                       name, symbol, decimals,
+                       issue_amount_wei,
+                       contract_registry_address,
+                       reserve_token_address,
+                       reserve_ratio_ppm):
+
+    return eth_manager.task_interfaces.composite.deploy_smart_token(
+        deploying_address,
+        name, symbol, decimals,
+        issue_amount_wei,
+        contract_registry_address,
+        reserve_token_address,
+        reserve_ratio_ppm)
+
+
+@celery_app.task(**base_task_config)
+def create_new_blockchain_wallet(self, wei_target_balance=0, wei_topup_threshold=0, private_key=None):
+    wallet = persistence_interface.create_new_blockchain_wallet(wei_target_balance=wei_target_balance,
+                                                                wei_topup_threshold=wei_topup_threshold,
+                                                                private_key=private_key)
     return wallet.address
 
 
-@celery_app.task(**task_config)
+@celery_app.task(**base_task_config)
 def topup_wallets(self):
-    higher_order_tasks.topup_wallets()
+    eth_manager.task_interfaces.composite.topup_wallets()
 
 
-@celery_app.task(**task_config)
-def topup_wallet_if_required(self, address, wei_target_balance, wei_topup_threshold):
-    higher_order_tasks.topup_if_required(address, wei_target_balance, wei_topup_threshold)
+@celery_app.task(**base_task_config)
+def topup_wallet_if_required(self, address):
```

Comment:
given these methods pass straight through to functions of with identical signatures, we could probably just add the celery decorator to the original functions, and refactor them out to be in an another celery tasks module. will do another time